gradlew

```diff
@@ -10,154 +10,163 @@
 # Resolve links: $0 may be a link
```

Comment:
Updating the `gradlew` isn't clean up. It should be separate and is called Updating Gradle Wrapper Script or something